techniques/system/common/1.0/rudder_agent_community_cron.st

```diff
@@ -9,4 +9,4 @@
 # /opt/rudder/etc/rudder-restart-message.txt file with your custom message inside. It will be sent by mail
```

Comment:
You could change the line above (touch /opt/rudder/etc/disable-agent) to say "run rudder agent disable" while you're here